README.rst

```diff
@@ -142,6 +143,35 @@ If you want to build "final" Docker images, then use:
     # GPU
     docker build -t preprod-pytorch:1.0.0-gpu-py3 -f docker/1.0.0/final/Dockerfile.gpu --build-arg py_version=3 .
 
+Amazon Elastic Inference with PyTorch in SageMaker
+------------------------------------------------
+`Amazon Elastic Inference <https://aws.amazon.com/machine-learning/elastic-inference/>`__ allows you to to attach
+low-cost GPU-powered acceleration to Amazon EC2 and Amazon SageMaker instances to reduce the cost of running deep
+learning inference by up to 75%. Currently, Amazon Elastic Inference supports TensorFlow, Apache MXNet, PyTorch,
+and ONNX models, with more frameworks coming soon.
+
+Support for using PyTorch with Amazon Elastic Inference in SageMaker is supported in the public SageMaker PyTorch serving containers.
+
+* For information on how Amazon Elastic Inference works, see `How EI Works <https://docs.aws.amazon.com/sagemaker/latest/dg/ei.html#ei-how-it-works>`__.
+* For more information in regards to using Amazon Elastic Inference in SageMaker, see `Amazon SageMaker Elastic Inference <https://docs.aws.amazon.com/sagemaker/latest/dg/ei.html>`__.
+
+Building the SageMaker Elastic Inference PyTorch Serving container
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Amazon Elastic Inference is designed to be used with AWS enhanced versions of TensorFlow serving, Apache MXNet or PyTorch serving.
+The SageMaker PyTorch containers with Amazon Elastic Inference support were built utilizing the
+same instructions listed `above <https://github.com/aws/sagemaker-pytorch-serving-container#building-your-image>`__ with the
```

Comment:
nit: Should this link just be `above <#building-your-image>` like in table of contents?